### DISCUSSION TITLE
Suggestions bind to keyboard shortcut

### DISCUSSION THREAD

**Discussion by alu0101330105** [community/community#14178] [Copilot Conversations]
### Suggestion for the copilot: 
It would be great if you could configure the copilot suggestions to appear after pressing a keyboard shortcut because sometimes its a bit distracting when huge blocks of text appears when you just wanna write some self-aid type of comment.

**Comment by zhuohoudeputao** [comment]
Yes, I agree with @alu0101330105 . When I just want a part of the suggestion, I want to use the "-->" key to select the part after where I type. For example,   when I type "What is ", it suggests "your name?", but I just want "your", then I want to use the "-->" key to select your, then I can just go on typing "job?".
